diff --git a/tools/linuxdeployqt/main.cpp b/tools/linuxdeployqt/main.cpp
index a1d5f12..eb2002e 100644
--- a/tools/linuxdeployqt/main.cpp
+++ b/tools/linuxdeployqt/main.cpp
@@ -193,6 +193,7 @@ int main(int argc, char **argv)
         qInfo() << "         https://github.com/probonopd/linuxdeployqt/issues/340";
      } else {
         // openSUSE Leap 15.0 uses glibc 2.26 and is used on OBS
+        /*
         if (strverscmp (glcv, "2.27") >= 0) {
             qInfo() << "ERROR: The host system is too new.";
             qInfo() << "Please run on a system with a glibc version no newer than what comes with the oldest";
@@ -202,6 +203,7 @@ int main(int argc, char **argv)
             qInfo() << "https://github.com/probonopd/linuxdeployqt/issues/340";
             return 1;
         }
+        */
     }
 
     if (argc < 2 || (firstArgument.startsWith("-"))) {
